OMG: 

```
/bin/ld: /var/cache/acbs/build/acbs.9olfuvx2/tg_owt/libtg_owt.so.0.0.0: undefined reference to `absl::raw_logging_internal::RawLog(absl::LogSeverity, char const*, int, char const*, ...)'
/bin/ld: /var/cache/acbs/build/acbs.9olfuvx2/tg_owt/libtg_owt.so.0.0.0: undefined reference to `absl::raw_logging_internal::internal_log_function[abi:cxx11]'
collect2: error: ld returned 1 exit status

```

--- a/extra-web/telegram-desktop/autobuild/prepare
+++ b/extra-web/telegram-desktop/autobuild/prepare
@@ -9,7 +9,7 @@ export CMAKE_PREFIX_PATH="$SRCDIR/../tg_owt:$CMAKE_PREFIX_PATH"
 
 abinfo "Making tg_owt..."
 cd "$SRCDIR"/../tg_owt
-cmake .
+cmake -DBUILD_SHARED_LIBS=OFF . 
 make
 
 cd "$SRCDIR"

